sysutils/libvirt/Portfile

```diff
@@ -135,10 +135,13 @@ variant sasl description {Use Cyrus SASL for authentication} {
     configure.args-replace  -Dsasl=disabled -Dsasl=enabled
 }
 
-variant qemu description {Enable the QEMU virtualization driver} {
-    depends_lib-append      port:qemu
-    configure.args-replace  -Ddriver_qemu=disabled -Ddriver_qemu=enabled
-}
+startupitem.create      yes
+startupitem.executable  ${prefix}/sbin/libvirtd
+
+startupitems        name        libvirt-system \
+                    location    LaunchDaemons \
+                    name        libvirt-session \
+                    location    LaunchAgents
```

Comment:
Does the user need to start both of these, or are you giving the user the choice of whether to start it as root or as their user account?

Reply:
It turns out the startup item wasn't working properly because it cannot find the qemu binary. The `${prefix}` isn't in the `PATH` when running `libvirtd` via launchd. I reverted the change for now as I'm not sure what's a nice way to solve it (the .plist does support `EnvironmentVariables` but it seems MacPorts does not expose that).